crates/fuzzing/src/oracles.rs

```diff
@@ -1331,6 +1332,22 @@ pub fn call_async(wasm: &[u8], config: &generators::Config, mut poll_amts: &[u32
     }
 }
 
+/// Fuzz target for table operations that reconstructs config from seed.
+pub fn fuzz_table_ops((config_seed, ops): (u64, generators::table_ops::TableOps)) {
+    let mut buf = [0u8; 1024];
+    let mut rng = rand::rngs::StdRng::seed_from_u64(config_seed);
+
+    for b in buf.iter_mut() {
+        *b = rng.r#gen()
+    }
+
+    let u = Unstructured::new(&buf);
+    let config = generators::Config::arbitrary_take_rest(u)
+        .expect("should be able to generate config from seed");
+
+    let _ = crate::oracles::table_ops(config, ops);
+}
+
```

Comment:
We already have an oracle for fuzzing table ops, which is what this function is ultimately calling. This is really just a small adapter for when we need to use `Arbitrary` to generate a config, and it feels out of place here. The only place we need to actually do that is in the fuzz target itself, so lets just inline it into the `fuzz_target!` invocation in `fuzz/fuzz_targets/table_ops.rs`.